app/controllers/users_controller.rb

```diff
@@ -318,8 +318,20 @@ def set_current_tab(current_tab = "profile")
     @tab = current_tab
   end
 
-  def config_changed?
-    params[:user].include?(:config_theme)
+  def less_than_one_day_old?(user)
```

Comment:
is this a leftover?

Reply:
no.

Reply:
so what is it used for? I can't find it used in this PR and it's not present in master 😕

Reply:
Ohh really sorry for this,  I think its leftover from conflict that I resolved. Will remove it if its not in master. Really sorry for this. Thanks!

Reply:
ahah no worries :) thanks for taking the time to look into this!